core/templates/pages/learner-dashboard-page/new-learner-dashboard.component.css

```diff
@@ -0,0 +1,371 @@
+/*
+  NOTE to developers: We generate RTL CSS from this file in the webpack
+  compilation, here are sme additional rules that can be added to the CSS files:
```

Comment:
Please fix the typo here: sme -> some.

No need to change it in other places as part of this PR.

Reply:
Done